api/src/pcapi/routes/backoffice_v3/venues.py

```diff
@@ -298,23 +300,70 @@ def get_venue_with_history(venue_id: int) -> offerers_models.Venue:
     return venue
 
 
-@venue_blueprint.route("/<int:venue_id>/details", methods=["GET"])
-def get_details(venue_id: int) -> utils.BackofficeResponse:
+@venue_blueprint.route("/<int:venue_id>/history", methods=["GET"])
+def get_history(venue_id: int) -> utils.BackofficeResponse:
     venue = get_venue_with_history(venue_id)
     actions = sorted(venue.action_history, key=lambda action: action.actionDate, reverse=True)
 
     form = forms.CommentForm()
     dst = url_for("backoffice_v3_web.venue.comment_venue", venue_id=venue.id)
 
     return render_template(
-        "venue/get/details.html",
+        "venue/get/history.html",
         venue=venue,
         actions=actions,
         dst=dst,
         form=form,
     )
 
 
+@venue_blueprint.route("/<int:venue_id>/invoices", methods=["GET"])
+def get_invoices(venue_id: int) -> utils.BackofficeResponse:
+    venue = (
+        offerers_models.Venue.query.filter(offerers_models.Venue.id == venue_id)
+        .options(
+            sa.orm.load_only(offerers_models.Venue.id),
+            sa.orm.joinedload(offerers_models.Venue.reimbursement_point_links)
+            .joinedload(offerers_models.VenueReimbursementPointLink.reimbursementPoint)
+            .load_only(offerers_models.Venue.id, offerers_models.Venue.name),
+        )
+        .one_or_none()
+    )
+
+    if not venue:
+        raise NotFound()
+
+    # Find current reimbursement point for the current venue, if different from itself
+    reimbursement_point = None
+    now = datetime.utcnow()
+    for reimbursement_link in venue.reimbursement_point_links:
+        if reimbursement_link.timespan.lower < now and (
+            not reimbursement_link.timespan.upper or reimbursement_link.timespan.upper > now
+        ):
+            if reimbursement_link.reimbursementPointId != venue.id:
+                reimbursement_point = reimbursement_link.reimbursementPoint
+            break
+
+    # Get invoices for the current venue as a reimbursement point
+    # We may have results even if the venue is no longer a reimbursement point
+    invoices = (
+        finance_models.Invoice.query.filter(finance_models.Invoice.reimbursementPointId == venue.id)
+        .options(
+            sa.orm.joinedload(finance_models.Invoice.cashflows)
+            .load_only(finance_models.Cashflow.batchId)
+            .joinedload(finance_models.Cashflow.batch)
+            .load_only(finance_models.CashflowBatch.label),
+        )
+        .order_by(finance_models.Invoice.date.desc())
+    ).all()
+
+    return render_template(
+        "venue/get/invoices.html",
+        reimbursment_point=reimbursement_point,
```

Comment:
(typo)

```suggestion
        reimbursement_point=reimbursement_point,
```

et aussi dans le template et dans le nom de quelques tests.